src/main/java/dc/Robot.java

```diff
@@ -193,22 +264,36 @@ public void robotPeriodic() {
   public void teleopInit() {
     System.out.println("Robot in operator control mode");
 
-    for (int i = 0; i < steerMotors.length; i++) {
-      //steerMotors[i].set(ControlMode.Position, ENCODER_OFFSETS[i] * 2048.0 / (Math.PI * 2.0 * STEER_RATIO));
-      steerMotors[i].set(ControlMode.Position, 0);
+    if (USE_MODULES) {
+    } else {
+      for (int i = 0; i < steerMotors.length; i++) {
+        //steerMotors[i].set(ControlMode.Position, ENCODER_OFFSETS[i] * 2048.0 / (Math.PI * 2.0 * STEER_RATIO));
+        steerMotors[i].set(ControlMode.Position, 0);
+      }
     }
   }
 
   @Override
   public void teleopPeriodic() {
-    drive.arcadeDrive(-stick.getY(), stick.getX());
+    if (USE_MODULES) {
+      double speed = -stick.getY();
+      frontLeftModule.set(speed, 0.0);
+      frontRightModule.set(speed, 0.0);
+      backLeftModule.set(speed, 0.0);
+      backRightModule.set(speed, 0.0);
```

Comment:
Why are we locking periodic to only the x axis? I feel like it would be useful to have full control of the robot to move it easily into position.

Reply:
Nevermind, forgot how this method worked.